corehq/apps/style/templates/style/includes/ko.html

```diff
@@ -14,5 +14,163 @@
 
 <script src="{% static 'style/js/components/inline_edit.js' %}"></script>
 <script src="{% static 'style/js/components/inline_edit_v2.js' %}"></script>
-<script src="{% static 'style/js/components.ko.js' %}"></script>
 {% endcompress %}
+
```

Comment:
Just curious - what's the advantage of putting this in an HTML file rather than a separate JS file?